internal/pkg/cli/validate.go

```diff
@@ -836,3 +850,14 @@ func validateSecretName(val interface{}) error {
 	}
 	return nil
 }
+
+func validateNotDockerfile(val interface{}) error {
+	s, ok := val.(string)
+	if !ok {
+		return errValueNotAString
+	}
+	if strings.HasSuffix(s, "Dockerfile") {
```

Comment:
Hmm does "a location has `Dockerfile` as suffix" equal to "they provide a local Dockerfile location"? I feel we could just update the prompt message...

Reply:
Sounds good, I've reverted the other changes and kept only the log. I agree it feels a bit unnecessary after the prompt msg update